docs/the_complex_matter_of_owning_work.md

```diff
@@ -0,0 +1,56 @@
+# @title The complex matter of owning work
```

Comment:
@title doesn't seem to render properly when I open this file in a new tab

Reply:
Its added syntax for yard to make sure it generates a sensible link name in the files list. Unfortunately it doesn't gel well with normal markdown in github and the like. Although in finding a link, I discovered that they now document how to hide the tags in GH:
https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#adding-meta-data-to-extra-files